excellent/functions/functions.go

```diff
@@ -41,51 +42,51 @@ var XFUNCTIONS = map[string]XFunction{
 	"legacy_add": LegacyAdd,
 
 	"round":      Round,
-	"round_up":   RoundUp,
-	"round_down": RoundDown,
+	"round_up":   OneNumberFunction("round_up", RoundUp),
```

Comment:
@nicpottier here's what functions look like using wrappers - can brainstorm if there's a better way to do this! Does suck having to repeat the function name so that's something to improve.